### PR TITLE
Adjust gitignore and remove lib folder from project

### PR DESCRIPTION
**Pull request by ginomessmer**
Removed due to faulty cached files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 target/
 bin/
+lib/
```
Comment (d0mmi) [inline .gitignore]: UwU

File: lib/apache-tomcat-9.0.21/webapps/DHBWProject_war/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -1,5 +0,0 @@
-Manifest-Version: 1.0
-Built-By: KE_BO
```
Comment (d0mmi) [inline lib/apache-tomcat-9.0.21/webapps/DHBWProject_war/META-INF/MANIFEST.MF]: OwO

File: lib/apache-tomcat-9.0.21/webapps/docs/WEB-INF/web.xml [hunk 3]
```diff
@@ -1,29 +0,0 @@
...
-  limitations under the License.
--->
-<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
-                      http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
-  version="4.0"
-  metadata-complete="true">
-
-  <display-name>Tomcat Documentation</display-name>
```
Comment (d0mmi) [inline lib/apache-tomcat-9.0.21/webapps/docs/WEB-INF/web.xml]: ;D

File: lib/apache-tomcat-9.0.21/webapps/DHBWProject_war/META-INF/MANIFEST.MF (hunk 2 above)
Comment (ginomessmer) [reply lib/apache-tomcat-9.0.21/webapps/DHBWProject_war/META-INF/MANIFEST.MF]: Very important

**Pull request merged**